src/pages/index.js

```diff
@@ -147,7 +138,7 @@ profileEditButton.addEventListener("click", openProfilePopup);
 profileImageButton.addEventListener("click", openChangeProfilePicturePopup);
 
 function openChangeProfilePicturePopup() {
-  changeProfilePictureFormValidator.toggleButtonState();
+  formValidators["change-profile-picture-form"].resetValidation();
```

Comment:
Good job

The avatar form has attribute `name="change-profile-picture-form"`, so everything works fine for it